barretenberg/cpp/src/barretenberg/polynomials/univariate.hpp

```diff
@@ -62,6 +62,17 @@ template <class Fr, size_t domain_end, size_t domain_start = 0> class Univariate
     const Fr& value_at(size_t i) const { return evaluations[i - domain_start]; };
     size_t size() { return evaluations.size(); };
 
+    // Check if the univariate is identically zero
```

Comment:
This allows a skip method of the form e.g. `in.q_arith.is_zero()` to work for both Univariate and FF (needed for perturbator). I was worried that checking ALL values in the Univariate would be expensive vs checking only the first 2 or 3 but it was not noticeable.